repositories.yaml

```diff
@@ -2028,3 +2028,25 @@ applications:
     channels:
       - v1_name: glam
         app_id: glam
+
+  - app_name: subscription_platform_backend
+    canonical_app_name: Subscription Platform
+    app_description: |
+      Mozilla Subscription Platform is Mozilla's subscription management
+      solution for account-based end-user services and features.
+    url: https://github.com/mozilla/fxa
+    notification_emails:
+      - dalvarez@mozilla.com
+    branch: main
+    metrics_files:
+      - libs/shared/metrics/glean/src/registry/subplat-backend-metrics.yaml
+    ping_files: []
+    dependencies:
+      - glean-server
+      - glean-js
+    moz_pipeline_metadata_defaults:
+      expiration_policy:
+        delete_after_days: 10000
```

Comment:
not sure if this is the right thing. It seems like the usual is to put a smaller default and then 10k for specific pings, but the bug did not specify. Should I ask?

Reply:
@edugfilho this is a good question for the product. I agree with you, 10k (27 years!?) seems like a weirdly high number. Maybe a typo?

Reply:
10k is probably a copy from some other app. These are default values that were put in when we started implementing finite retention and effectively serve as placeholders for indefinite retention. I'd recommend [760 days to keep it consistent with fxa](https://github.com/mozilla/probe-scraper/blob/4373ec1137e17e03337d78edcec42c28a150b6c6/repositories.yaml#L1631).